src/modules/app/components/StatusBar.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import * as React from 'react'
+import type { ThemeType } from '../../theme/constants/theme'
+import { StatusBar as ReactNativeStatusBar } from 'react-native'
+
+type PropsType = {|
+  theme: ThemeType
+|}
+
+class StatusBar extends React.PureComponent<PropsType> {
+  render () {
+    return (
+      <ReactNativeStatusBar backgroundColor={this.props.theme.colors.backgroundAccentColor} barStyle='dark-content' />
```

Comment:
I don' really remember what barStyle does here :/ 
But actually I think it makes sense that the text it always is black.

Reply:
It does exactly what the goal of the Issue was. Notify Android/iOS that the text is dark, and therefore the status bar has to be white.
From the docs `barStyle

Sets the color of the status bar text.`

Reply:
I've found options for theming the app (`brightColors`, `darkColors`; theme.js) but had no access at changing it with the JS files. What I think is, that if `darkColors` is a real option for choosing at the CMS (I quite have no access to the CMS at the moment), the static value `dark-content` of barStyle would lead into a misconfiguration.

Reply:
@lulo4907 I don't really understand your point. :thinking: All colors are defined on our side, we don't get anything from the cms.

Reply:
@lulo4907 is right about the theming problem, right now the dark theme is not used. Maybe we should remove it for now until we want to add a dark mode.

Reply:
As long it is not used I don't see a problem? At least I don't really see a point in removing it now.

Reply:
We just have to keep an eye on this upon implementing a dark mode.

Reply:
Created an issue with a note to keep this in mind: https://issues.integreat-app.de/browse/NATIVE-526